clients/go/admin/pkce/auth_flow_orchestrator.go

```diff
@@ -0,0 +1,166 @@
+package pkce
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/service"
+	"github.com/flyteorg/flytestdlib/logger"
+
+	"github.com/pkg/browser"
+	"golang.org/x/oauth2"
+)
+
+const (
+	Timeout     = 15 * time.Second
```

Comment:
should this be configurable?

Reply:
I was on the fence... alright, will move it to config